docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -471,7 +471,7 @@ public class PersonResource {
 
     @GET
     @Path("/{id}")
-    public Person get(Long id) {
+    public Person get(@PathParam Long id) {
```

Comment:
RESTEasy Reactive (which is what we recommend and what's used in this guide) shouldn't require the annotation. Did you have actual issues?

Reply:
Yes, I had issue using the RESTEasy classic https://github.com/tarilabs/demo20220716/blob/907f9e842edc86add34df4f31decc6f0711a24eb/pom.xml#L37

My bad, I missed the previous paragraph mentioned RESTEasy reactive 🙏